src/core/shape_inference/include/strided_slice_shape_inference.hpp

```diff
@@ -4,10 +4,11 @@
 
 #pragma once
```

Comment:
Why did you move `sequnce_generator.hpp` to developer API?

Reply:
At some point I've done it to avoid adding shape inference lib dependency to `ngraph_test_util` library.
But it looks like is not required now so move this file back.